recipes-devtools/rust/rust.inc

```diff
@@ -117,11 +117,13 @@ python do_configure() {
 
     config.set("build", "vendor", e(True))
 
-    targets = [d.getVar("TARGET_SYS", True)]
+    if not "targets" in locals():
+        targets = [d.getVar("TARGET_SYS", True)]
     config.set("build", "target", e(targets))
 
-    hosts = [d.getVar("HOST_SYS", True)]
-    config.set("build", "host", e(targets))
+    if not "hosts" in locals():
+        hosts = [d.getVar("HOST_SYS", True)]
+    config.set("build", "host", e(hosts))
```

Comment:
Looks like this was an existing copy/paste bug.  Nice!

Reply:
Yeah, it was revealed during the factorization, that's how the code duplication elimination pays off itself :)

Thanks!